app/resources/api/v2/transfer_template_resource.rb

```diff
@@ -30,6 +30,12 @@ class TransferTemplateResource < BaseResource
       # @!attribute [r] uuid
       #   @return [String] the UUID of the transfer template.
       attribute :uuid, readonly: true
+
+      # @!method filter_uuid
+      #   Filter the transfer templates by UUID.
+      #   @example URL with UUID filter
+      #     https://sequencescape.psd.sanger.ac.uk/api/v2/transfer_templates?filter[uuid]=12345678-1234-1234-1234-123456789012
```

Comment:
I think we try to avoid putting the repo urls on GitHub, maybe just put `https://<sequencescape>/api/v2/transfer_templates?filter[uuid]=12345678-1234-1234-1234-123456789012`

Reply:
Good idea.  It's a bit long as it is anyway.